From a7d53cb38bdea0ad29bb5c1e430638130e41ba57 Mon Sep 17 00:00:00 2001
From: Peter Bittner <django@bittner.it>
Date: Sat, 12 Dec 2020 19:46:29 +0100
Subject: [PATCH] Adapt Tox target for building the docs

This will generate the HTML docs in the same location as `invoke docs`.
---
 tox.ini | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/tox.ini b/tox.ini
index b825921..8ccb58b 100644
--- a/tox.ini
+++ b/tox.ini
@@ -77,12 +77,9 @@ setenv =
 
 
 [testenv:docs]
-basepython= python2
 changedir = docs
-commands=
-    sphinx-build -W -b html -D language=en -d {envtmpdir}/doctrees . {envtmpdir}/html/en
-deps=
-    -r{toxinidir}/py.requirements/docs.txt
+commands = sphinx-build -W -b html -D language=en -d {toxinidir}/build/docs/doctrees . {toxinidir}/build/docs/html/en
+deps = -r{toxinidir}/py.requirements/docs.txt
 
 
 [testenv:cleanroom2]
@@ -146,4 +143,3 @@ commands=
 deps=
     jit
     {[testenv]deps}
-
